README.md

```diff
@@ -2053,6 +2053,50 @@ Finished in 0.5 seconds
 27 tests, 0 failures
 ```
 
+### 11.4 Add Turbolinks
+
+Add [turbolinks package](https://www.npmjs.com/package/turbolinks) to `/assets/package.json`:
+
+```sh
+cd assets && npm install turbolinks --save
+```
+
+In `assets/app.js` import turbolinks:
+
+```js
+import Turbolinks from "turbolinks"
+```
+
+and start it by calling the `start` function at the end of the `app.js` file:
+
+```js
+Turbolinks.start();
+```
+
+Finally move the `app.js` script call from the html body to the header in `lib/app_web/tempaltes/layout/app.html.eex`:
```

Comment:
@SimonLab does the script _need_ to be in the `<head>` ? (_given that Turbolinks will only run after the DOM has fully loaded, I would argue that it makes sense to have it as far down in the page as possible to avoid blocking the content load_)